tests/node-18.0/tests.js

```diff
@@ -1,37 +1,71 @@
 const fetch = require("node-fetch");
 
-/*
-    'req' variable has:
-        'headers' - object with request headers
-        'payload' - object with request body data
-        'variables' - object with function variables
-    'res' variable has:
-        'send(text, status)' - function to return text response. Status code defaults to 200
-        'json(obj, status)' - function to return JSON response. Status code defaults to 200
-    
-    If an error is thrown, a response with code 500 will be returned.
-*/
+module.exports = async (context) => {
+    const action = context.req.headers['x-action'];
 
-module.exports = async (req, res) => {
-    const payload = JSON.parse(req.payload === '' ? '{}' : req.payload);
+    switch (action) {
```

Comment:
We should also be able to support the `text/html` content type. And we need some tests for that

Reply:
Added support for `context.res.html('<h1>Your HTML code</h1>)`